pkg/runtime/runtime.go

```diff
@@ -960,7 +961,11 @@ func (a *DaprRuntime) initPubSub(c components_v1alpha1.Component) error {
 	}
 
 	properties := a.convertMetadataItemsToProperties(c.Spec.Metadata)
-	properties["consumerID"] = a.runtimeConfig.ID
+	consumerID := strings.TrimSpace(properties["consumerID"])
+	if consumerID == "" {
+		consumerID = a.runtimeConfig.ID
```

Comment:
Can we somehow account for this line in unit tests?

Reply:
Done.